mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapper.java

```diff
@@ -150,7 +152,9 @@ private SimpleMailbox mailbox(Row row) {
             row.getLong(UIDVALIDITY));
         mailbox.setMailboxId(CassandraId.of(row.getUUID(ID)));
         mailbox.setACL(new CassandraACLMapper(mailbox, session, maxRetry).getACL());
+
         return mailbox;
+        
```

Comment:
This file does not need to change so I reset to apache/master.
But, yes Sir. I won't use the git command line to commit the file, I will synchronize carefully when do the commit.